packages/cell2location/meta.yaml

```
@@ -0,0 +1,25 @@
+name: cell2location
+description: |
+    Cell2location is a Bayesian model that can resolve fine-grained cell types in spatial transcriptomic data 
+    and create comprehensive cellular maps of diverse tissues. Cell2location accounts for technical sources 
+    of variation and borrows statistical strength across locations, thereby enabling the integration of single-cell 
+    and spatial transcriptomics with higher sensitivity and resolution than existing tools.
+project_home: https://github.com/BayraktarLab/cell2location
+documentation_home: https://cell2location.readthedocs.io/en/latest/
+tutorials_home: https://cell2location.readthedocs.io/en/latest/
+publications:
+    - 10.1038/s41587-021-01139-4
+install:
+    pypi: cell2location
+tags:
+    - Bayesian
+license: Apache-2.0
+version: v0.1
+authors:
+    - vitkl
+    - yozhikoff
+    - mayerdev
+    - emdann
+    - Alexander Aivazidis
+test_command: |
+    pip install ".[test]" && pytest
```

```suggestion
    pip install ".[dev]" && pytest
```

see https://github.com/BayraktarLab/cell2location/blob/ebd924873f0327c59af1c7146efb5575145b5b7e/.github/workflows/test.yml#L39